### PR TITLE
Incoming avoidance

### PR DESCRIPTION
**Pull request by liamaxelrod**
Will add the features of the car to back up and move forward when an obstacle gets to close
 
This pull request is linked to issue #22 
and fulfills the first four criteria of the issue

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [ ] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: arduino/smartcar/smartcar.ino [hunk 1]
```diff
@@ -11,9 +11,13 @@
 MQTTClient mqtt;
 WiFiClient net;
 
+bool safetyFeatures = true;
 bool canDrive = true;
 bool safetySystem;
 
+bool activeAvoidance = false;
+//bool inMotion = false;
```
Comment (Valkama1) [inline arduino/smartcar/smartcar.ino:18]: I think we should remove code that is commented out or explain why it's there in the future.

**Review by Valkama1 — DISMISSED** [review]
Couldn't find any other issues, looks good Liam

**Review by Valkama1 — APPROVED** [review]
already approved idk whats going on

**Comment by liamaxelrod** [comment]
Not all the criteria was fulfilled so I closed the pull request for the time being

**Pull request closed without merging**